docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -72,7 +75,10 @@ nuget config -set repositoryPath=c:\packages -configfile %ProgramFiles(x86)%\NuG
 Mac/Linux:
 
 ```cli
-# Set repositoryPath in the user-level config file
+# Set globalPackagesFolder (available for PackageReference only) in the user-level config file
+nuget config -set globalPackagesFolder=/home/packages
+
+# Set repositoryPath (available for packages.config only) in the user-level config file
 nuget config -set repositoryPath=/home/packages 
```

Comment:
Same comment as before. repositoryPath should not be set at the user level, and even if it sometimes works, I really don't feel like we should be "encouraging" it with examples like this.